.readthedocs.yaml

```diff
@@ -32,4 +32,6 @@ sphinx:
 # See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
 python:
   install:
-    - requirements: docs/api_sphinx/requirements.txt
\ No newline at end of file
+    - requirements: docs/api_sphinx/requirements.txt
```

Comment:
Nit on the directory. Let's match the intention

Reply:
Did we want to put this inside a separate `api-docs` directory that's not managed by gitbook?

Reply:
Maybe - I was hoping it would kinda magically work under gitbooks but maybe that's a bad idea. If so we should start a `python/api_reference` directory to leave room for js.